crm_claim_type/demo/crm_claim.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" ?>
+<odoo>
+    <record id="crm_claim.crm_claim_1" model="crm.claim">
```

Comment:
demo data should be on tests

Reply:
You can include demo data in your module as well, but they are only for demo purpose. In fact, a good practice is to include some demo data that showcases your module.

Reply:
ok